lib/index.html

```diff
@@ -74,23 +73,23 @@ <h2>Design and engineering, joined together</h2>
 			<section class="feature-tiles">
 				<div class="feature">
 					<div class="tile">
-						<img src="./images/wireframe.png" />
+						<img src="./images/wireframe.png" alt="Image of a wireframe" />
```

Comment:
Is the word "image" not a bit redundant here?

Reply:
That's right. Should we just use alt="" and role="presentation"?

Reply:
I think the whole feature tiles section will not make any sense to someone that is visually impaired.
You will just hear:
```
"... and replace all the placeholder components."
"Wireframe"
"Design"
"Design & Code"
```

I think the complete section is just visual presentation